code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -79,6 +79,9 @@
 	icon_living = "mouse_[mouse_color]"
 	icon_dead = "mouse_[mouse_color]_dead"
 	icon_resting = "mouse_[mouse_color]_sleep"
+	set_mouse_desc()
+
+/mob/living/simple_animal/mouse/proc/set_mouse_desc()
```

Comment:
is there any particular reason why this has to be abstracted into its own proc?

Reply:
![image](https://user-images.githubusercontent.com/77684085/196980873-825a63dd-dd77-471d-bf2b-9d90cd5e6646.png)
Doing like this is better? I have been suggested to make a proc but maybe it isnt the best option.